### PR TITLE
Feature/attach screenshots allure report

### PR DESCRIPTION
**Pull request by goncer**

### CODE REVIEW

File: utils/frontend/utils/Helper.ts [hunk 1]
```diff
@@ -125,6 +125,8 @@ export async function addExtraLogs(driver: WebDriver, testName = "") {
   });
   const img = await driver.takeScreenshot();
   fs.writeFileSync(`${outputPath}/screenshot_${testName}.png`, img, "base64");
+  const reporter = (globalThis as any).reporter as Reporter;
+  reporter.addAttachment("Screeenshot", new Buffer(img, "base64"), "image/png");
 }
```
Comment (goncer) [inline utils/frontend/utils/Helper.ts:130]: those are the important changes

File: .github/workflows/recurrent-checks.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: CI check
 
 on:
   schedule:
-    - cron: "1 01,13 * * *" #lets run this job every 12 hours.
+    - cron: "1 01 * * *" #lets run this job every 12 hours.
```
Comment (CodeByEddy) [inline .github/workflows/recurrent-checks.yml:5]: This is every day at 01:01 (I think!). Can we update either the cron to be every 12 hours or the comment to say it runs nightly?

**Review by CodeByEddy — APPROVED** [review]

**Comment by CodeByEddy** [comment]
Approved with a minor change request (so my review isn't blocking the PR)

File: .github/workflows/recurrent-checks.yml (hunk 2 above)
Comment (goncer) [reply .github/workflows/recurrent-checks.yml:5]: true

**Pull request merged**